lib/ramble/ramble/workspace/workspace.py

```diff
@@ -281,17 +281,17 @@ def active_workspace():
 
 def get_workspace_path():
     """Returns current directory of ramble-managed workspaces"""
-    path_in_config = ramble.config.get('config:workspace_dirs')
+    path_in_config = ramble.config.get('config:workspace_dirs', scope='user')
     if not path_in_config:
-        path_in_config = '$ramble/var/ramble/GET_WORKSPACE_ERROR/'
+        path_in_config = ramble.config.get('config:workspace_dirs', scope='_builtin')
```

Comment:
If we get the correct definition is above, and path is not set in the config, I think this should be considered an error. In that case, we should call `tty.die` instead to kill the program.

Reply:
changed as requested